test/powershell/Modules/PSReadLine/PSReadLine.Tests.ps1

```diff
@@ -42,6 +42,13 @@ Describe "PSReadLine" -tags "CI" {
         (Get-PSReadLineKeyHandler | where { $_.Key -ceq "Alt+B" }).Function | Should Be SelectBackwardWord
     }
 
+    It "Should ignore case when using Function binding" {
+        Set-PSReadlineOption -EditMode Windows
+        $lowerCaseFunctionName = "yank";
+        Set-PSReadlineKeyHandler "Ctrl+k" -Function $lowerCaseFunctionName
```

Comment:
I guess setting the EditMode interferes in the same way? I should probably revert after.